docs/agents/appmotion_motor.rst

```diff
@@ -0,0 +1,224 @@
+.. highlight:: rst
+
+.. _appmotion_motor:
+
+============================
+Applied Motion Motors Agent
+============================
+
+This agent is used to communicate with NEMA motors, particularly the beam-
+mapper motors.
+
+.. argparse::
+   :filename: ../agents/appmotion_motor/appmotion_motors_agent.py
+   :func: make_parser
+   :prog: python3 appmotion_motors_agent.py
+
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+OCS Site Config
+````````````````
+
+An example site-config-file block::
+
+    {'agent-class': 'appMotionMotorsAgent',
+         'instance-id': 'beammap-motors',
+         'manage': 'docker',
+         'arguments':[
+           ['--motor1-ip', '192.168.0.188'],
+           ['--motor1-port', '4001'],
+           ['--motor1-is-lin'],
+           ['--motor2-ip', '192.168.0.188'],
+           ['--motor2-port', '4002'],
+           #['--motor2-is-lin'],
+           ['--sampling-frequency','2'],
+           ['--mode', 'acq'],
+         ]},
+
+Docker Compose
+``````````````
+
+An example docker-compose configuration::
+
+    ocs-beam-mapper:
+    <<: *ocs-base
+    image: motor_test_image
+    network_mode: "host"
+    environment:
+      LOGLEVEL: debug
+    command:
+      - "--instance-id=beammap-motors"
```

Comment:
`image: motor_test_image` -> `image: simonsobs/ocs-app-motion-motors-agent`

Also, this entire block needs to be tabbed over underneath `ocs-beam-mapper:`.